data/mods/gen4/random-teams.ts

```diff
@@ -22,9 +22,9 @@ export class RandomGen4Teams extends RandomGen5Teams {
 				!counter.get('Bug') && (movePool.includes('bugbuzz') || movePool.includes('megahorn'))
 			),
 			Dark: (movePool, moves, abilities, types, counter) => (
-				(!counter.get('Dark') || (counter.get('Dark') === 1 && moves.has('pursuit'))) &&
-				movePool.includes('suckerpunch') &&
-				counter.setupType !== 'Special'
+				((!counter.get('Dark') || (counter.get('Dark') === 1 && moves.has('pursuit'))) &&
+				movePool.includes('suckerpunch') && counter.setupType !== 'Special') ||
+				(!counter.get('Dark') && types.has('Poison'))
```

Comment:
> Approximately 16% (114 sets) had the move Poison Jab and did not have the move Crunch.

this is intended? if not, this condition can be simplified

Reply:
Yes, it is intended to allow Drapion to have Pursuit and no Crunch. Dark STAB enforcement has just been improved so it should look a lot cleaner now.